Collection.js

```diff
@@ -12,26 +12,29 @@ Invoices.byTimeRange = function(filter, sortBy, sortOrder,limit){
   }
   sortQuery[sortBy] = sortOrder;
 
-  let curDate = new Date();
+  let endDate = new Date();
   let startDate = new Date();
+  let daywrapper = moment(startDate);
```

Comment:
you can just refactor `daywrapper` to be `let startDate = moment()` -> this is already the same as `moment(new Date())` :wink: